inc/timberDynamicResize.php

```diff
@@ -2,247 +2,32 @@
 
 namespace Flynt\TimberDynamicResize;
 
+use Flynt\Utils\TimberDynamicResize;
 use Twig\TwigFilter;
-use Timber;
 
-const DB_VERSION = '1.1';
-const TABLE_NAME = 'resized_images';
-const IMAGE_ROUTE = 'dynamic-images';
-const IMAGE_PATH_SEPARATOR = 'dynamic';
+// Disable dynamic images and fall back to regular Timber resize filter
+// add_filter('Flynt/TimberDynamicResize/disable', __return_true);
 
-add_filter('init', 'Flynt\TimberDynamicResize\registerRewriteRule');
+// Disable WebP generation
+// add_filter('Flynt/TimberDynamicResize/disableWebP', __return_true);
```

Comment:
Are there issues with web image generation, or is it just an option added?

Reply:
if you use a cdn in front of your page or some other sort of caching, generating webp or rather adding the htaccess rewrite rules can lead to issues. if you then do not need the webp images, they should not be generated because of performance and storage size reasons.